package.json

```diff
@@ -4,28 +4,24 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "npm run init && npm-run-all --parallel js:watch js:watch:local css:watch css:watch:local kibble:watch kibble:watch:local json:kibble:lang:watch",
-    "css:init": "npm-run-all css:local:backup css:local:variables css:local:custom",
-    "init": "node tools/build-output.js --mode=init && npm-run-all css:init css js merge:json:kibble merge:json:lang",
+    "start": "npm run init && npm-run-all --parallel js:watch js:watch:local css:watch kibble:watch kibble:watch:local json:kibble:lang:watch",
+    "init": "node tools/build-output.js --mode=init && npm-run-all css js merge:json:kibble merge:json:lang",
     "build": "npm run kibble:build",
-    "publish": "npm run build",
```

Comment:
Can we keep this alias? This is the one I use and what I recommend to clients to use. The reason being in my mind "publish" is more akin to deploying the site, while "build" doesn't carry that connotation - for me I would expect a "build" to be a local thing only.

It also ties to `npm publish` (to publish a package).

In fact, I would be happy if `kibble:build` didn't have the `&& kibble publish` and we separated them out, then `publish` could be `npm run kibble:build && kibble publish`.